command/docker.go

```diff
@@ -61,14 +62,14 @@ func (d *DockerManager) ComposeFileUp(composeFilePath string, opts ...pulumi.Res
 
 	return d.runner.Command(
 		d.namer.ResourceName("run", composeFilePath),
-		pulumi.Sprintf("docker-compose -f %s up --detach --wait --timeout 300", remoteComposePath),
-		nil,
-		pulumi.Sprintf("docker-compose -f %s down -t 300", remoteComposePath),
-		nil, true,
+		&CommandArgs{
+			Create: pulumi.Sprintf("docker-compose -f %s up --detach --wait --timeout 300", remoteComposePath),
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement. -->
**🥜 nitpick**
Did you mean 
```suggestion
			Create: pulumi.Sprintf("docker-compose -f %s up --detach --wait --timeout %d", remoteComposePath, defaultTimeout),
```
?